src/app/tracing/tracing.actions.ts

```diff
@@ -105,6 +107,17 @@ export class FocusGraphElementSSA implements Action {
     constructor(public payload: { elementId: NodeId | EdgeId }) {}
 }
 
+export class RefreshLastUnchangedJsonDataExtractMSA implements Action {
```

Comment:
Please use `SetLastUnchangedJsonDataExtractMSA`instead of `RefreshLastUnchangedJsonDataExtractMSA`

Reply:
is renamed!
Also `RefreshLastUnchangedJsonDataExtractSuccessSOA` is renamed to `SetLastUnchangedJsonDataExtractSuccessSOA`